index.js

```diff
@@ -0,0 +1,52 @@
+import Books from './modules/input.js';
+import displayDateTime from './modules/display_date.js';
+import Navigater from './modules/navigater.js';
+
+const a = new Books();
+const lists = document.getElementById('Lists');
+lists.addEventListener('click', (e) => {
+  e.preventDefault();
+  const display = new Navigater();
+  display.lists();
+});
+
+const aBooks = document.getElementById('AddBooks');
+aBooks.addEventListener('click', (e) => {
+  e.preventDefault();
+  const display = new Navigater();
+  display.addNewBook();
+  display.saveToLocalStorage();
+  window.location.reload();
+});
+
+const contactus = document.getElementById('contactuss');
+contactus.addEventListener('click', (e) => {
+  e.preventDefault();
+  const display = new Navigater();
+  display.contactus();
+});
+
+const formbut = document.getElementById('addbtn');
```

Comment:
- [ ] As said early, kindly make sure you give your variable a more descriptive name for readability and maintenance of your code

Reply:
- [ ] I think it will be more professional to change this variable name to a more descriptive name. 👍 Descriptive variable names really make reading code a lot easier. ℹ️